src/components/forms/KeyResultValueForm.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <form-section>
+    <form-component
+      v-model.number="value"
+      input-type="input"
+      name="value"
+      :label="$t('keyResult.newValue')"
+      rules="required"
+      type="number"
+    />
+
+    <form-component
+      v-model="comment"
+      input-type="textarea"
+      name="comment"
+      :rows="commentRows"
+      :label="$t('keyResult.addComment')"
+      :placeholder="$t('keyResult.commentPlaceholder')"
+    />
+
+    <template #actions="{ handleSubmit, submitDisabled }">
+      <btn-save :disabled="submitDisabled || loading" @click="handleSubmit(submitForm)" />
+    </template>
+  </form-section>
+</template>
+
+<script>
+import { FormSection, BtnSave } from '@/components/generic/form';
+
+export default {
+  name: 'AddKeyResultValueForm',
+
+  components: {
+    FormSection,
+    BtnSave,
+  },
+
+  props: {
+    keyResult: {
+      type: Object,
+      required: true,
+    },
+    commentRows: {
+      required: false,
+      type: Number,
+      default: 4,
+    },
```

Comment:
Hvorfor ha denne? Ser vi for oss varianter av denne med et annet antall rader snart?

Reply:
Jeg gjorde en `:comment-rows="2"` i `KeyResultHome.vue` for å få skjemaet en smule mer kompakt, eller tenkte du på noe annet?

Reply:
Ja, kunne vi ikke bare fjernet prop'en og satt `rows=2` her hvis det er den eneste høyden vi ser for oss å trenge? ;)

Reply:
Sorry, skjønner ikke helt: Du tenker å fjerne muligheten for å konfigurere det via `FormComponent`, eller kun for dette skjemaet? Vi bruker jo fortsatt `rows=4` der den brukes ellers (`KeyResultModal`)?

Reply:
```diff
diff --git a/src/components/forms/KeyResultValueForm.vue b/src/components/forms/KeyResultValueForm.vue
index e50a88a3..1b034f57 100644
--- a/src/components/forms/KeyResultValueForm.vue
+++ b/src/components/forms/KeyResultValueForm.vue
@@ -13,7 +13,7 @@
       v-model="comment"
       input-type="textarea"
       name="comment"
-      :rows="commentRows"
+      rows=2
       :label="$t('keyResult.addComment')"
       :placeholder="$t('keyResult.commentPlaceholder')"
     />
@@ -40,11 +40,6 @@ export default {
       type: Object,
       required: true,
     },
-    commentRows: {
-      required: false,
-      type: Number,
-      default: 4,
-    },
     loading: {
       required: false,
       type: Boolean,
diff --git a/src/views/KeyResultHome.vue b/src/views/KeyResultHome.vue
index 469fa84f..af87b170 100644
--- a/src/views/KeyResultHome.vue
+++ b/src/views/KeyResultHome.vue
@@ -44,7 +44,6 @@
           <key-result-value-form
             :key-result="activeKeyResult"
             :loading="isSaving"
-            :comment-rows="2"
             @save="saveProgress"
           />
         </div>
```

Reply:
Hadde ikke tenkt til å ha 2 rader i KeyResultModal, men kan godt det.

Reply:
Ah, så den brukes flere steder ja. Det var det jeg lurte på og ikke så. Da er kommentarene mine her moot. :p